centinel/primitives/dnslib.py

```diff
@@ -15,7 +15,7 @@ def get_ips(host, nameserver=None, record="A"):
     return lookup_domain(host, nameservers=nameservers, rtype=record)
 
 
-def lookup_domain(domain, nameservers=[], rtype="A", timeout=10):
+def lookup_domain(domain, nameservers=[], rtype="A", timeout=2):
```

Comment:
Though I understand that we could still extract useful information from a pcap, I'd still argue that having one clean pcap for each URL makes the results easier to use later. A giant pcap that contains data for ~1000 URLs is going to be unwieldy and unnecessary. Uploading one big pcap file is also more challenging in poor network conditions than a uploading one small pcap at a time.